_develop/connect/use-with-clarity.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Usage with Clarity
+{:.no_toc}
+
+With Connect, you can interact with the Stacks 2.0 blockchain. You can allow your users to send transactions and interact with smart contracts.
+
+* TOC
+{:toc}
+
+## How it works
+
+For your app's users to be able to execute a smart contract function, they need to sign and broadcast a transaction. It's important that users remain control of the private keys that sign these transactions. Connect provides an easy-to-use workflow that allows your users to securely sign transactions.
```

Comment:
maybe an animation/preview would help here? I think that would make it tangible

Reply:
Agreed - @markmhx can we turn that demo into a Gif or something?

Reply:
Sounds like a good idea generally, but which demo do you have in mind exactly?

Reply:
I was thinking the demo from the forum post that introduces Secret Key